flow/layers/layer_tree_unittests.cc

```diff
@@ -198,7 +199,8 @@ TEST_F(LayerTreeTest, PrerollContextInitialization) {
   // PrerollContext that this test must be revisited and updated.
   // If any fields get removed or replaced, then the expect_defaults closure
   // will fail to compile, again bringing attention to updating this test.
-  EXPECT_EQ(sizeof(PrerollContext), size_t(104));
+  EXPECT_EQ(sizeof(PrerollContext),
+            size_t(104) + sizeof(std::vector<RasterCacheItem*>*));
```

Comment:
You can just update the size_t number instead of adding in the sizeof. This test just makes us aware that the size changed in case someone causes its size to change by accident.